__tests__/ExpensiMark-HTML-test.js

```diff
@@ -1470,3 +1470,29 @@ test('Test autoEmail with markdown of <pre>, <code>, <a>, <mention-user> and <em
 
     expect(parser.replace(testString)).toBe(resultString);
 });
+
+test('UserMention', () => {
+    let testString = '@user@domain.com';
+    expect(parser.replace(testString)).toBe('<mention-user>@user@domain.com</mention-user>');
+
+    testString = '@USER@DOMAIN.COM';
+    expect(parser.replace(testString)).toBe('<mention-user>@user@domain.com</mention-user>');
+
+    testString = '@USER@domain.com';
+    expect(parser.replace(testString)).toBe('<mention-user>@user@domain.com</mention-user>');
+
+    testString = '@user@DOMAIN.com';
+    expect(parser.replace(testString)).toBe('<mention-user>@user@domain.com</mention-user>');
+
+     testString = '@user@domain.com';
+    expect(parser.replace('<mention-user>@user@domain.com</mention-user>')).toBe(testString);
+
+    testString = '@USER@DOMAIN.COM';
+    expect(parser.replace('<mention-user>@USER@DOMAIN.COM</mention-user>')).toBe(testString);
+
+    testString = '@USER@domain.com';
+    expect(parser.replace('<mention-user>@USER@domain.com</mention-user>')).toBe(testString);
+
+    testString = '@user@DOMAIN.com';
+    expect(parser.replace('<mention-user>@user@DOMAIN.com</mention-user>')).toBe(testString);
```

Comment:
This file is for testing markdown to html, not the other way around. Let's remove these tests and keep them in the proper file.